hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -679,6 +697,18 @@
       name: chip_clkmgr_external_clk_src
       desc: '''Verify the clkmgr switches to the correct clk src during certain LC states.
 
+            On POR lc asserts lc_clk_byp_req, and de-asserts it when lc_program
+            completes. It may be best to verify this via SVA, unless we
+            implement clock cycle counters.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_clkmgr_jitter_enable
+      desc: '''Verify the chip operates correctly when clock jitter is enabled.
+
+            Run a full regression with the chip configured with jittery clocks.
```

Comment:
@cfrantz, @moidx, @tjaychen 
In which SW stage is clk jitter enabled?

Reply:
As early as possible. I suspect this is something we can define behind a sw OTP and configure in the mask ROM.

Reply:
Since the DV tests run with a custom boot ROM, perhaps we could replicate that setting in there. Should be easy to control via SW knob whether to turn this on / off for specific DV tests.

Reply:
Added mention of using the DV custom boot ROM and knobs to control jitter.